core/src/test/java/com/dooapp/fxform/issues/Issue150Test.java

```diff
@@ -119,6 +121,7 @@ public void test150() throws ExecutionException, InterruptedException {
         thread1.start();
         thread2.start();
         Thread.sleep(1000);
+        running.set(false);
```

Comment:
I updated this issue to stop the threads at the end because when I run all tests from the console I saw some ConcurrentModificationException (with no consequence) into the logs.
It was because these threads were still running.